Gems/Atom/RPI/Code/Source/RPI.Public/Scene.cpp

```diff
@@ -34,6 +35,11 @@ namespace AZ
         ScenePtr Scene::CreateScene(const SceneDescriptor& sceneDescriptor)
         {
             Scene* scene = aznew Scene();
+            scene->m_name = sceneDescriptor.m_nameId;
+
+            AZ::Name visSceneName(AZStd::string::format("RenderCullScene[%s]", scene->m_name.GetCStr()));
```

Comment:
You can avoid a memory allocation here by using `AZStd::fixed_string<64>::format` instead